src/Build/Construction/Solution/SolutionProjectGenerator.cs

```diff
@@ -2325,9 +2325,11 @@ private void AddGlobalProperties(ProjectRootElement traversalProject)
             ProjectPropertyElement tfv20Property = frameworkVersionProperties.AddProperty("TargetFrameworkVersion", "v2.0");
             ProjectPropertyElement tfv35Property = frameworkVersionProperties.AddProperty("TargetFrameworkVersion", "v3.5");
             ProjectPropertyElement tfv40Property = frameworkVersionProperties.AddProperty("TargetFrameworkVersion", "v4.0");
+            ProjectPropertyElement tfv48Property = frameworkVersionProperties.AddProperty("TargetFrameworkVersion", "v4.8");
```

Comment:
Are the changes in this file necessary? I wouldn't expect them to be and would love to not update product code in this PR at all.